packages/yoga/src/DropdownInput/web/DropdownInput.jsx

```diff
@@ -0,0 +1,317 @@
+          ${dropdowninput.container.border.width}px;
+        border-radius: ${dropdowninput.container.border.radius}px
+          ${dropdowninput.container.border.radius}px 0 0;
+      `}
+  `}
+`;
+
+const Clear = styled.button`
+  ${({
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    background: ${dropdowninput.clear.background};
+    border: ${dropdowninput.clear.border};
+    cursor: ${dropdowninput.clear.cursor};
+    display: ${dropdowninput.clear.display};
+    outline:  ${dropdowninput.clear.outline};
+    padding: ${dropdowninput.clear.padding}
+  `}
+`;
+
+const Input = styled.input`
+  background: none;
+  border: none;
+  color: ${theme.colors.text.primary};
+  flex: 1;
+  font-family: ${({ theme: { yoga } }) => yoga.baseFont.family};
+  font-size: ${theme.fontSizes.small}px;
+  font-style: normal;
+  font-weight: ${theme.fontWeights.regular};
+  height: 100%;
+  letter-spacing: 0px;
+  line-height: ${theme.lineHeights.small}px;
+  outline: none;
+  text-align: left;
+  padding-left: ${theme.spacing.small}px;
+
+  &::placeholder {
+    color: ${theme.colors.text.secondary};
+  }
+
+  ${({ value }) =>
+    value &&
+    css`
+      + ${Clear} {
+        display: block;
+      }
+    `}
+`;
+
+const Divisor = styled.div`
+  width: 1px;
+  height: 28px;
+  background: ${theme.colors.elements.lineAndBorders};
+`;
+
+const ContainerDropDown = styled.ul`
+  ${({
+    theme: {
+      yoga: {
+        components: { dropdowninput },
+      },
+    },
+  }) => `
+    margin: 0;
+    height: 272px;
+    width: ${props => `${props.width}px`};
+    top: ${props => `${props.position}px`};
+    overflow-y: scroll;
+    border-left: ${dropdowninput.containerDropDown.border.width} solid ${
+    dropdowninput.containerDropDown.border.color
+  };
+    border-right: ${dropdowninput.containerDropDown.border.width} solid ${
+    dropdowninput.containerDropDown.border.color
+  };
+    border-bottom: ${dropdowninput.containerDropDown.border.width} solid ${
+    dropdowninput.containerDropDown.border.color
+  };
+    list-style: none;
+    padding: 0 ${dropdowninput.containerDropDown.padding.right}px 0 ${
+    dropdowninput.containerDropDown.padding.left
+  }px;
+    transition: all 0.1s ease-in-out;
+    border-radius: 0 0 ${dropdowninput.containerDropDown.border.radius}px ${
+    dropdowninput.containerDropDown.border.radius
+  }px;
+    background: ${dropdowninput.containerDropDown.background};
+    position: absolute;
+    z-index: 9999;
+  `}
+`;
+
+const ButtonDropDown = styled.button`
+  display: flex;
+  align-items: center;
+  width: auto;
+  background: none;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  padding: 0;
+  height: 100%;
+`;
+
+const ItemDropDown = styled.li`
+  ${display}
+  ${flexes}
+  width: 100%;
+  height: 72px;
+  cursor: pointer;
+
+  div {
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const ContainerName = styled.div``;
+
+const DropdownInput = ({
```

Comment:
The name of this component is too the generic, considering we have this one:

https://gympass.github.io/yoga/components/dropdown

So, WDYT of?
```suggestion
const CountryDropdownInput = ({
```